tests/xilinx_test.py

```diff
@@ -40,6 +40,11 @@
     ("tests/fpga/streaming_memory.py", "streamingcomp_1", True, True, []),
     ("tests/fpga/conflict_resolution.py", "fpga_conflict_resolution", True,
      False, []),
+    # This doesn't pipeline with Vitis 2020.1 for whatever reason (it pipelines
+    # with both 2019.2 and 2020.2), so just switch this back on once CI starts
+    # running 2020.2 or newer.
+    ("tests/transformations/mapfusion_fpga.py",
```

Comment:
can you add this also to Intel?

Reply:
It's already there!

Reply:
In fact, Intel is the reason why we needed this fix in the first place :-)

Reply:
Gotcha!